blivet/devices/md.py

```diff
@@ -195,6 +198,14 @@ def level(self, value):
 
         self._level = level
 
+    @property
+    def sector_size(self):
+        if not self.exists:
+            # Least common multiple of parents' sector sizes
+            return reduce(lambda a, b: a * b // math.gcd(a, b), [p.sector_size for p in self.parents])
```

Comment:
Could use a generator instead of list comprehension, right? :stuck_out_tongue:

Reply:
Yes, fixed.